package.json

```diff
@@ -33,15 +33,16 @@
   "dependencies": {
     "ansi-align": "^2.0.0",
     "camelcase": "^4.0.0",
-    "chalk": "^2.0.1",
+    "chalk": "^2.1.0",
```

Comment:
Revert it back. Semver handles this.

Reply:
This came from upgrading the packages. Not intentionally.